samples/subsys/llext/fs_loader/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024 Schneider Electric
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(fs_loader)
+
+target_sources(app PRIVATE src/main.c src/log_extension.c src/export.c)
+
+# declare extension
+zephyr_llext(hello_world)
+zephyr_llext_sources(hello_world.llext/hello_world.c)
+
+zephyr_llext(hello_world_pic PIC)
+zephyr_llext_sources(hello_world.llext/hello_world.c)
+
+set(FAT_IMG ${CMAKE_CURRENT_BINARY_DIR}/llext_vfat.img)
+
+# generate image
+add_custom_target( vfat.img ALL
+    COMMAND ${CMAKE_CURRENT_LIST_DIR}/makeimg.sh ${FAT_IMG} ${CMAKE_CURRENT_BINARY_DIR}/hello_world.llext ${CMAKE_CURRENT_BINARY_DIR}/hello_world_pic.llext
+    WORKING_DIRECTORY ${ZEPHYR_BASE}
+    SOURCES ${CMAKE_CURRENT_LIST_DIR}/makeimg.sh
+    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/hello_world.llext ${CMAKE_CURRENT_BINARY_DIR}/hello_world_pic.llext
+)
+
+if(${CONFIG_BOARD} STREQUAL "nucleo_l496zg")
+    #flash fs image
+    add_custom_target( flash_fs_jlink
+        COMMAND openocd -f interface/jlink.cfg -c "transport select swd" -f target/stm32l4x.cfg  -c init -c "reset halt" -c "flash write_image erase ${FAT_IMG} 0x080E0000" -c "reset" -c shutdown
+        WORKING_DIRECTORY ${ZEPHYR_BASE}
+    )
+    add_custom_target( flash_fs_stlink
+        COMMAND openocd -f interface/stlink-v2.cfg -f target/stm32l4x.cfg -c init -c "reset halt" -c "flash write_image erase ${FAT_IMG} 0x080E0000" -c "reset" -c shutdown
+        WORKING_DIRECTORY ${ZEPHYR_BASE}
+    )
```

Comment:
this is a no

Reply:
This is a terrible review comment and entirely unhelpful, please guide the PR author in the right direction

Reply:
Well, I removed it. (even if I think it is quite developer friendly to have an example and be able to use it with West)